contents/tutorials/limit-session-recordings.md

```diff
@@ -162,6 +172,31 @@ This is useful if you want to record specific parts or paths on a page such as:
 - new or updated features
 - smaller, specific components within a larger component.
 
+# Sampling
+
+You can configure sampling to limit the number of sessions you record for each user. This is useful if you want to record a percentage of sessions for all users. Sampling helps reduce the number of sessions you record, but it doesn’t let you control which sessions are included.
+
+Our recommendation is to start with no sampling (e.g. 100% - the default) or a low sampling rate (e.g. 90% or 95%) and increase it as needed. This helps you get a sense of how many sessions you’re recording and how much data you’re collecting.
+
+
+![sampling config shown set to 100% i.e. no sampling](../images/tutorials/limit-session-recordings/sampling-config.png)
+
+Whenever a new session starts after you have selected a collection below 100% the browser will generate a random number between 0 and 1. If the number is less than the sampling rate (e.g. 0.9 when set to 90%), the session will be recorded. If it is greater than the sampling rate, the session will not be recorded.
+
+# Minimum duration
+
+You can also set a minimum duration for sessions to be recorded. This is useful if you want to exclude sessions that are too short to be useful. For example, you might want to exclude sessions that are less than 2 seconds long to avoid recording sessions where users quickly bounce off your site.
+
+![minimum duration config shown set to 2 seconds](../images/tutorials/limit-session-recordings/min-duration.png)
+
+The minimum duration is set in seconds. Whenever a new session starts, the browser records the start time. When deciding whether to send recording data to the backend if the minimum duration has passed since the start time, the session data will be sent. If it hasn't, the session will continue to be buffered in-memory. 
```

Comment:
```suggestion
The minimum duration is set in seconds. Whenever a new session starts, the browser records the start time. If the minimum duration has passed since the start time, the session data is sent. If it hasn't, the session continues to be buffered in-memory. 
```